test/e2e/density.go

```diff
@@ -190,8 +190,10 @@ var _ = Describe("Density [Skipped]", func() {
 	for _, testArg := range densityTests {
 		name := fmt.Sprintf("should allow starting %d pods per node", testArg.podsPerNode)
 		if testArg.podsPerNode == 30 {
-			name = "[Performance] " + name
+			name = "[Feature:Performance] " + name
 			framework.addonResourceConstraints = density30AddonResourceVerifier()
+		} else {
+			name = "[Skipped] " + name
```

Comment:
We have some predefined types of density tests: 3, 30 and 100. 3 and 100 are for manual runs only and we don't want to run them on Jenkins, as they either don't provide any value (3), or are above our current SLO for density (100). It's still useful to have them defined to make running them easy (I use density 3 pretty often to check various stuff, and density 100 is/was run by RedHat and/or Samsung @smarterclayton @spiffxp). I add a flag to define density, but it'll probably make test scripts even worse. 

We can change Skipped to Manual to better reflect the meaning. WDYT?